scripts/benchmark_inference.py

```diff
@@ -225,7 +225,8 @@ def one_token(model, use_cache):
     if local_rank == 0 and not args.skip_correctness_check:
         torch.testing.assert_close(actual, expected)
     else:
-        torch.cuda.synchronize()
+        if torch.cuda.is_available():
```

Comment:
should we check device_type=='cuda' instead of cuda available? if we use cpu on a host where cuda is available, we still don't need to synchronize

Reply:
Do we want to pass device_type in the argument of this function or check `model.device()`?

Reply:
we might want to re-think how we're handling globals in functions here at some point, but the script has a global `args.device_type` (and model does not have a `device()`)

Reply:
Thanks, updated the PR.